platformio.ini

```diff
@@ -28,6 +28,11 @@ config_gps = -DUSE_GPS -DGPS_RX_PIN=34 -DGPS_TX_PIN=12
 config_lora = -DRFM95_CS=18 -DRFM95_RST=23 -DRFM95_INT=26
 config_gps = -DUSE_GPS -DGPS_RX_PIN=12 -DGPS_TX_PIN=15
 
+; TTGO LORA V1.2 (T-FOX)
+[ttgo_lora_v12]
```

Comment:
to not confuse them with the other lora 32 modules maybe adding `t-fox` similar to the `t-beam` might be a good idea

Reply:
I've renamed the env's as well as the settings section in 97fdc0c, as requested.